ftfy/char_classes.py

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
Very nitpicky thing, but: do you want to rerun build_data.py, just to get the comment at the start of this file?